src/humanloop/otel/exporter.py

```diff
@@ -128,265 +110,64 @@ def force_flush(self, timeout_millis: int = 10000) -> bool:
         return True
 
     def _do_work(self):
-        """Upload spans to Humanloop.
-
-        Ran by worker threads. The threads use the self._shutdown flag to wait
-        for Spans to arrive. Setting a timeout on self._upload_queue.get() risks
-        shutting down the thread early as no Spans are produced e.g. while waiting
-        for user input into the instrumented feature or application.
-
-        Each thread will upload a Span to Humanloop, provided the Span has all its
-        dependencies uploaded. The dependency happens in a Flow Trace context, where
-        the Trace parent must be uploaded first. The Span Processor will send in Spans
-        bottoms-up, while the upload of a Trace happens top-down. If a Span did not
-        have its span uploaded yet, it will be re-queued to be uploaded later.
-        """
```

Comment:
should still replace this with a correct docstring

Reply:
outdated